runtime/util/modularityHelper.c

```diff
@@ -211,7 +211,7 @@ isPackageExportedToModuleHelper(J9VMThread *currentThread, J9Module *fromModule,
 		isExported = TRUE;
 	} else if (NULL != j9package) {
 		/* First try the general export rules */
-		BOOLEAN const isExportedAll = j9package->exportToAll || (toUnnamed ? j9package->exportToAllUnnamed : FALSE);
+		BOOLEAN const isExportedAll = (1 == j9package->exportToAll) || (toUnnamed ? (1 == j9package->exportToAllUnnamed) : FALSE);
```

Comment:
Please say ` (0 != j9package->exportToAll)` instead of ` (1 == j9package->exportToAll)`.

Reply:
Updated.